edgelet/contrib/debian/postrm

```diff
@@ -3,22 +3,32 @@ set -e
 
 case "$1" in
     purge)
-        if [ -f /etc/aziot/edged/config.toml ]; then
-            rm /etc/aziot/edged/config.toml
-        fi
-        if [ -d /etc/aziot/edged/config.d ]; then
-            rm -r /etc/aziot/edged/config.d
-        fi
-        if [ -d /var/lib/aziot/edged ]; then
-            rm -rf /var/lib/aziot/edged
+        systemctl daemon-reload
```

Comment:
Why do you need this?

Reply:
This clears the uninstalled package from commands like systemctl status, etc.